### PR TITLE
Update Importing 3D scenes documentation for Godot 4.0

### PR DESCRIPTION
**Pull request by Calinou**
Follow-up to https://github.com/godotengine/godot-docs/pull/6782 (can be merged independently).

- Document the Advanced Import Settings dialog for 3D scenes.
- Replace occurrences of "DCC" with "3D modeling software" across the documentation, which is more universally understood.

### CODE REVIEW

File: tutorials/assets_pipeline/importing_scenes.rst [hunk 1]
```diff
@@ -188,327 +188,538 @@ the scene will lead to more consistent results and should be done whenever
...
+  in the 3D scene. If set to **Static Lightmaps**, sets the meshes' GI mode to
+  **Static** and generates UV2 on import for :ref:`lightmap baking <doc_using_lightmap_gi>`.
+- **Lightmap Texel Size:** Only visible if **Light Baking** is set to **Static
+  Lightmaps**. Controls the size of each texel on the baked lightmap. A smaller
+  value results in more precise lightmaps, at the cost of larger lightmap sizes
+  and longer bake times.
+
+**Skins**
+
+- **Use Named Skins:** If checked, use named skins for animation.
```
Comment (Calinou) [inline tutorials/assets_pipeline/importing_scenes.rst:291]: I don't understand what named skins are for exactly. I remember this having to do with animation reuse on different unrelated models. Can someone help me fill this description?

File: tutorials/assets_pipeline/importing_scenes.rst [hunk 2]
```diff
@@ -188,327 +188,538 @@ the scene will lead to more consistent results and should be done whenever
...
-FPS
-~~~
-
-Most 3D export formats store animation timeline in seconds instead of frames. To ensure
-animations are imported as faithfully as possible, please specify the frames per second
-used to edit them. Failing to do this may result in shaky animations.
-
-Filter Script
-~~~~~~~~~~~~~
+Filter script
```
Comment (Calinou) [inline tutorials/assets_pipeline/importing_scenes.rst:590]: Can someone confirm this feature was removed in 4.0 as part of the import system rewrite? I don't see any references to it in the UI.

**Comment by mhilbrunner** [comment]
@Calinou Needs a rebase :)

File: tutorials/assets_pipeline/importing_scenes.rst (hunk 1 above)
Comment (lyuma) [reply tutorials/assets_pipeline/importing_scenes.rst:291]: Okay, so let's start with explaining what a Skin object is. Actually let's go back more and explain what the role of a Skeleton3D node is, and how it relates to Mesh rendering.

The MeshInstance3D node contains three properties of relevance: a skeleton node path, pointing to the Skeleton3D node, usually "..", a mesh, and a skin:

- The Skeleton3D node contains a list of bones with names, their pose and rest, a name and a parent bone.
- The mesh is all of the raw vertex data needed to display a mesh. In terms of the mesh, it knows how vertices weight painted, and uses some internal numbering often imported from a Digital Content Creation (DCC) tool such as Blender.
- The skin contains the information necessary to bind this mesh onto this Skeleton3D. For every one of the internal bone ids chosen by the DCC tool, it contains two things. Firstly, a Matrix known as the Bind Pose Matrix, Inverse Bind Matrix, or IBM for short. Secondly, the Skin contains each bone's name (if the Use Named Binds setting is on by default), or the bone's index within the Skeleton3D list (if Use Named Binds is turned off).

Together, this information is enough to tell Godot how to use the bone poses in the Skeleton3D node to render the mesh from each MeshInstance3D. Note that each MeshInstance3D may share binds, as is common in models exported from Blender, or each MeshInstance3D may use a separate Skin object, as is common in models exported from other tools such as Maya.

Comment (Calinou) [reply tutorials/assets_pipeline/importing_scenes.rst:291]: Thanks a lot! I've added this information to the page :slightly_smiling_face:

**Pull request merged**